### DISCUSSION TITLE
Kong Manager in Kong OSS

### DISCUSSION THREAD

**Discussion by pritishpattanaik** [Kong/kong#12296] [Help]
Hi All, 

How to secure Kong Manager ?  I am running kong 3.5 on RHEL with hybrid deployment. Please advise if anyone has configured authentication in Kong Manager. Did anyone encounter this challenge. I don't understand what's the point of having a beautiful Admin UI without authentication. I just proposed  my team to use Kong OSS but now i am completely stuck. 

So far i've created two services 

1. AdminAPI Service with http://<kong_control_plane_ip>:8001 
    
   route - /admin-api

3. KongManager service with http://<kong_control_plane_ip>:8002

   route -   /manager 
               ~/asset.*      # added for accessing css/html 

I am getting below error while accessing Kong manager via data plane route. 

http://<kong_data_plane_ip>:8000/manager 

error in chrome console. 

Refused to apply style from 'http://10.x.x.x:8000/assets/index-518a32f1.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

**Comment by guanlan** [comment]
similar discussion: https://github.com/Kong/kong/discussions/12072
